### PR TITLE
Support checked and unchecked

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

**Comment by Liaojinghui** [comment]
Cool

**Review by Liaojinghui — APPROVED** [review]

**Comment by erikzhang** [comment]
@devhawk What do you think of this PR?

File: src/Neo.Compiler.CSharp/MethodConvert.cs [hunk 1]
```diff
@@ -2053,26 +2068,78 @@ private void ConvertBinaryExpression(SemanticModel model, BinaryExpressionSyntax
...
-                "&" => OpCode.AND,
-                "^" => OpCode.XOR,
-                "==" => OpCode.EQUAL,
-                "!=" => OpCode.NOTEQUAL,
-                "<" => OpCode.LT,
-                "<=" => OpCode.LE,
-                ">" => OpCode.GT,
-                ">=" => OpCode.GE,
+            (OpCode opcode, bool checkResult) = expression.OperatorToken.ValueText switch
+            {
```
Comment (shargon) [inline src/Neo.Compiler.CSharp/MethodConvert.cs:2072]: https://github.com/neo-project/neo-devpack-dotnet/blob/00011c6b5ced0cf1a797fe6ceace9006cdd83306/src/Neo.Compiler.CSharp/MethodConvert.cs#L3896 check range in Pow?

Comment (erikzhang) [reply src/Neo.Compiler.CSharp/MethodConvert.cs:2072]: The `checked` and `unchecked` keywords do not affect method calls.

**Review by devhawk — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Pull request merged**